<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainComendant</class>
 <widget class="QMainWindow" name="MainComendant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Режим коменданта</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="dormitory_groupbox">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Общежитие № 0</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="address_label">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Адрес:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="comendant_label">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Комендант: </string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="building_1_label">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Корпус № 0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="room_amount_1_label">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Количество комнат:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="place_amount_1_label">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Количество мест в комнате:</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="building_2_label">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Корпус № 0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="room_amount_2_label">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Количество комнат:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="place_amount_2_label">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Количество мест в комнате:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>Редактирование данные об общежитии.</string>
            </property>
            <property name="text">
             <string>Редактировать</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="state_label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>10</height>
       </size>
      </property>
      <property name="text">
       <string>Комендант: Главная</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="toolTip">
     <string>Редактирование данных о лицах, проживающих в общежитии.</string>
    </property>
    <property name="title">
     <string>Управление общежитием</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="toolTip">
     <string>Получение данных.</string>
    </property>
    <property name="title">
     <string>Данные</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="toolTip">
     <string>Получение списка заявок.</string>
    </property>
    <property name="title">
     <string>Заявки</string>
    </property>
    <addaction name="action_7"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="toolTip">
     <string>Доступ к справке.</string>
    </property>
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Добавить информацию о заселенных лицах</string>
   </property>
   <property name="toolTip">
    <string>Добавить информацию о заселенных лицах</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Изменить информацию о выселенных лицах</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Получить список свободных мест</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Получить список свободных комнат</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Получить информацию о комнате</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Получить список проживающих</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Получить список заявок</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Справка пользователя</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Справка разработчика</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
